settings.yml

```diff
@@ -39,3 +39,4 @@ zuul:
     trigger: 'timer'
     project: 'acid-test-dev'
     policy: 'RejectPolicy'
+  gearman_conf: ''
```

Comment:
Rebase your branch for new configuration locations and structure.
Add `gearman_conf` parameter inside `zuul` (like you did here).
Add comment description of what it is, what it does, and what should be put here.